gnucash/python/init.py

```diff
@@ -9,7 +9,7 @@
 import os
 import gettext
 # install gettext for _-function, needs localepath
-gettext.install("gnucash",
+gettext.install(_sw_core_utils.GETTEXT_PACKAGE,
         _sw_core_utils.gnc_path_get_localedir())
```

Comment:
I wonder if you can/want define shorthands for _sw_core_utils, but that has low priority.